### DISCUSSION TITLE
[Question]: Frigate detector strategy

### DISCUSSION THREAD

**Discussion by philippedegeer** [blakeblackshear/frigate#10838] [Ask A Question]
### What is your question:

Where are we in terms of Frigate running Yolov8 in future versions ?  I have seen in other discussions that "support" would be removed and it is not clear to me what it means. I have also noticed a change in the wording of the documentation.

Will the interfacing code be removed ?

I am asking as I have invested a considerable amount of time in training Yolov8 for my personal need, with great results when it comes to detection accuracy and false positives. It would be a bummer to lose it all.

Can the Frigate owners/contributors (thanks for the outstanding work) clarity what the long term strategy for detectors will be ?

**Comment by NickM-27** [comment]
You can see https://github.com/blakeblackshear/frigate/pull/10717

That is all that is known currently
